devices/Button/ButtonHoldingState.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Iot.Device.Button
+{
+    public enum ButtonHoldingState
```

Comment:
Please add the Intellisense comments. Yes, these might look obvious, still...

Reply:
Good catch, added!